lib/logitech_receiver/descriptors.py

```diff
@@ -143,7 +143,7 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Keyboards
 
 _D('Wireless Keyboard K230', protocol=2.0, wpid='400D')
-_D('Wireless Keyboard K270', protocol=1.0,
+_D('Wireless Keyboard K270', protocol=1.0, wpid='4023',
```

Comment:
Please, check if your K270 is really HID++ 1.0. I have one with HID++ 2.0, we cannot mix both at this moment.